src/test/java/oshi/SystemInfoTest.java

```diff
@@ -46,6 +46,13 @@ public void testGetMemory() {
         assertTrue(memory.getAvailable() <= memory.getTotal());
     }
 
+    @Test
+    public void testCpuLoad() {
+        SystemInfo si = new SystemInfo();
+        HardwareAbstractionLayer hal = si.getHardware();
+        System.out.println(hal.getProcessorLoad());

```

Comment:
This isn't a test, it just prints it out. What we want is something that at least checks that the output is a non-zero % number for example.